randovania/games/prime1/json_data/header.json

```diff
@@ -1755,9 +1756,10 @@
                         },
                         "lock": null
```

Comment:
the requirement and lock need to be updated for this and for bomb doors to reflect their conversion to blast shields. requirement should be the same as blue doors, and lock should be the old requirement